apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.html.ts

```diff
@@ -151,3 +107,194 @@ export function InstructorAssessmentAccess({
     </html>
   `.toString();
 }
+
+function EditAccessRuleModal({
+  accessRule,
+  timeZoneName,
+}: {
+  accessRule: AssessmentAccessRules;
+  timeZoneName: string;
+}) {
+  return Modal({
+    id: `editAccessRuleModal`,
+    title: '',
+    body: html`
+      <div class="form-group">
+        <input type="hidden" name="row" class="access-rule-row" />
+        <label for="mode">Mode</label>
+        <select
+          class="form-control access-rule-mode"
+          id="mode"
+          name="mode"
+          aria-describedby="modeHelp"
+        >
+          <option value="" ${accessRule.assessment_access_rule.mode === null ? 'selected' : ''}>
+            —
+          </option>
+          <option
+            value="Exam"
+            ${accessRule.assessment_access_rule.mode === 'Exam' ? 'selected' : ''}
+          >
+            Exam
+          </option>
+          <option
+            value="Public"
+            ${accessRule.assessment_access_rule.mode === 'Public' ? 'selected' : ''}
+          >
+            Public
+          </option>
+        </select>
+        <small id="modeHelp" class="form-text text-muted">
+          Used to restrict access to assessments based on the current mode. In general, it is best
+          to use "Public" for any homework and "Exam" for exams proctored with PrairieTest. (See the
+          <a
+            href="https://prairielearn.readthedocs.io/en/latest/accessControl/#server-modes"
+            target="_blank"
+            >documentation</a
+          >
+          for more information.)
```

Comment:
Nit, this doesn't have to be a parenthetical.

```suggestion
          to use "Public" for any homework and "Exam" for exams proctored with PrairieTest. See the
          <a
            href="https://prairielearn.readthedocs.io/en/latest/accessControl/#server-modes"
            target="_blank"
            >documentation</a
          >
          for more information.
```